concept/error.rs

```diff
@@ -162,6 +165,9 @@ pub enum ConceptReadError {
     SnapshotGet { source: SnapshotGetError },
     SnapshotIterate { source: Arc<SnapshotIteratorError> },
     Encoding { source: EncodingError },
+    CannotGetLabelForExistingType,
```

Comment:
It's like extreme errors of "it has to exist, but for some reason it doesn't" instead of `unwrap`s

Reply:
If so, let's call this `CorruptMissingLabelOfType` or something extreme :)

Reply:
Cool!